File: java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/common/api/Booking.java
package io.oasp.application.mtsj.bookingmanagement.common.api;

import java.sql.Timestamp;

import io.oasp.application.mtsj.bookingmanagement.common.api.datatype.BookingType;
import io.oasp.application.mtsj.general.common.api.ApplicationEntity;

public interface Booking extends ApplicationEntity {

  public String getName();

  public void setName(String name);

  public String getBookingToken();

  public void setBookingToken(String bookingToken);

  public String getComment();

  public void setComment(String comment);

  public Timestamp getBookingDate();

  public void setBookingDate(Timestamp bookingDate);

  public Timestamp getExpirationDate();

  public void setExpirationDate(Timestamp expirationDate);

  public Timestamp getCreationDate();

  public void setCreationDate(Timestamp creationDate);

  public String getEmail();

  public void setEmail(String email);

  public Boolean getCanceled();

  public void setCanceled(Boolean canceled);

  public BookingType getBookingType();

  public void setBookingType(BookingType bookingType);

  public Long getTableId();

  public void setTableId(Long tableId);

  public Long getOrderId();

  public void setOrderId(Long orderId);

  public Integer getAssistants();

  public void setAssistants(Integer assistants);

  public Long getUserId();

  public void setUserId(Long userId);

}

File: net/netcore/OASP4Net.XUnitTest/Model/Booking.cs
using System;
using System.Collections.Generic;

namespace OASP4Net.Test.Model
{
    public partial class Booking
    {
        public Booking()
        {
            InvitedGuest = new HashSet<InvitedGuest>();
            Order = new HashSet<Order>();
        }

        public long Id { get; set; }
        public long? UserId { get; set; }
        public string Name { get; set; }
        public string ReservationToken { get; set; }
        public string Comments { get; set; }
        public string BookingDate { get; set; }
        public string ExpirationDate { get; set; }
        public string CreationDate { get; set; }
        public string Canceled { get; set; }
        public long? IdBookingType { get; set; }
        public long? TableId { get; set; }
        public string Email { get; set; }
        public long? Assistants { get; set; }

        public Table Table { get; set; }
        public User User { get; set; }
        public ICollection<InvitedGuest> InvitedGuest { get; set; }
        public ICollection<Order> Order { get; set; }
    }
}
